### ISSUE TITLE
Question, how to create signing package in sandboxed environment on SmartOS pkgbuild zone

### ISSUE THREAD

**Issue by drscream** [joyent/pkgbuild#6]
Based on the documentation in the [README.md](https://github.com/joyent/pkgbuild/blob/master/README.md) and on the [wiki](https://github.com/joyent/pkgsrc/wiki/pkgdev:signing) I try to get signing enabled for an sandboxed environment in the pkgbuild zone. But mostly I run into the issue that the `GPG` variable and `GPG_SIGN_AS` is not configured in the `pkg_install.conf` environment.

Example output from an sandbox:

```
--<root@pkgbuild-16q4>-(/data/chroot/dev-2016Q4-x86_64)-</data/pkgsrc/www/fcgi>--
-> bmake package
=> Bootstrap dependency digest>=20010302: found digest-20160304
WARNING: [license.mk] Every package should define a LICENSE.
=> Creating binary package /home/pbulk/build/www/fcgi/work/.packages/fcgi-2.4.0nb2.tgz
=> Signing binary package /home/pbulk/build/www/fcgi/work/.packages/fcgi-2.4.0nb2.tgz (GPG)
pkg_admin: GPG variable not set
*** Error code 1
```

Based on the documentation I've also created the files like:

`/data/pkgbuild/conf/{{ pkgsrc_release}}/mk.conf.local`:

```
SIGN_PACKAGES=  gpg
```

`/data/pkgbuild/conf/{{ pkgsrc_release }}/pkgbuild.conf.local`:

```
PKGSRC_GPG_SIGN_AS=_MY_ID_
```

After I've modified the `pkg_install.conf` in the environment everything is working correctly. I hope you could help me to setup the environment correctly.

Thanks and best regards,
Thomas

**Comment by jperkin** [comment]
Yeh, sorry, this is still work-in-progress, and yeh at the moment you need to hardcode the `pkg_install.conf` settings. I'll see about ensuring the `run-sandbox` script modifies the `pkg_install.conf` unpacked into the chroot based on the pkgbuild settings or something.

**Comment by jperkin** [comment]
Fixed by joyent/pkgbuild#7
